pkgs/development/python-modules/coremltools/default.nix

```diff
@@ -0,0 +1,77 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, attrs
+, numpy
+, packaging
+, protobuf
+, scipy
+, six
+, sympy
+, tqdm
+, onnx
+, pandas
+, pillow
+, pytest
+, pytorch
+, scikitlearn
+, torchvision
+}:
+
+buildPythonPackage rec {
+  pname = "coremltools";
+  version = "4.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "579897ca1ce33384ebc0cebc4e43a2e946104168279ec171f8710d6485c42acc";
+  };
+
+  propagatedBuildInputs = [
+    attrs
+    numpy
+    packaging
+    protobuf
+    scipy
+    six
+    sympy
+    tqdm
+  ];
+
+  # attr should not be there
+  # the correct name is attrs (which is already in the list)
+  prePatch = ''
+    substituteInPlace setup.py \
+      --replace '"attr",' ""
+  '';
+
+  checkInputs = [
+    onnx
+    pandas
+    pillow
+    pytest
+    pytorch
+    scikitlearn
+    torchvision
+  ];
+
+  # skip tests which require downloaded models or macOS
+  checkPhase = ''
+    pytest -k "not (
```

Comment:
Please use pytestCheckHook together with disabledTests.

Reply:
Addressed in f7777b8c133b821e5098dae6a317be7ca2ebf724